code/modules/modular_computers/computers/subtypes/preset_silicon.dm

```diff
@@ -3,9 +3,9 @@
 
 /obj/item/modular_computer/silicon/preset/install_default_hardware()
 	. = ..()
-	processor_unit = new/obj/item/computer_hardware/processor_unit(src)
-	hard_drive = new/obj/item/computer_hardware/hard_drive(src)
-	network_card = new/obj/item/computer_hardware/network_card(src)
+	processor_unit = new /obj/item/computer_hardware/processor_unit(src)
+	hard_drive = new /obj/item/computer_hardware/hard_drive(src)
+	network_card = new /obj/item/computer_hardware/network_card/advanced(src)
```

Comment:
Maybe we could have preset special for AI with wired card? Might be out of scope

Reply:
Out of scope.